pages/common/irb.md

```diff
@@ -0,0 +1,11 @@
+# irb
+
+> Interactive Ruby Shell.
+
+- Run Ruby Sell with load some gem:
```

Comment:
Please start with the simplest invocation of the command, in this case `irb`. Keep the description as `Launch a REPL(Read-Eval-Print-Loop) shell`